### ISSUE TITLE
About detecting many extra boxes

### ISSUE THREAD

**Issue by pengjizhi** [open-mmlab/mmdetection#1773]
Hello ！
Drawing on your ideas, I am developing my own single-stage object detection network, but there are a lot of extra boxes in the visualization results.
Since my network structure is deep, I guess it may be caused by the disappearance of the gradient or the explosion of the gradient, but why do these two situations lead to the appearance of a large number of abnormal boxes, can you give any suggestions?

In addition, I don't think this is the cause of nms. In fact, such results are already the results after nms.

![lindau_000000_000019_detection](https://user-images.githubusercontent.com/40751283/70404783-104a7680-1a76-11ea-894e-15328aeb83f3.png)

**Comment by ZwwWayne** [comment]
You might need to modify the nms threshold.

**Comment by pengjizhi** [comment]
@ZwwWayne 
Hello there! Thank you for your enthusiastic answers!
Let me explain my training situation. I started the training with 0.001 as the initial learning rate and saved the results every 10 epochs. However, this abnormal situation occurred when I used a weight test of 60 epochs and the picture above The situation, that is, a large number of abnormal boxes, but using the weight of the 50th epoch or earlier weights to test, the result is normal.

For your answer,
1. When the model does not show an abnormal boxes, the map behaves normally. When the abnormal boxes appears, the map is greatly reduced. Usually,  the abnormal boxes appears in the second half of the training.
2. The threshold of visualization is 0.5. As shown in the figure, the model predicts a large number of boxes with thresholds exceeding 0.5. I really cannot understand this situation.

**Comment by ZwwWayne** [comment]
Then it seems the model is not trained so well after the 50th epoch.
Do you decay the learning after 50 between 60 epochs? Or this is because of some other reasons resulting from your data? I am not sure.

**Comment by pengjizhi** [comment]
@ZwwWayne 
Indeed, as you said, the reason for this situation is due to poor training, if I reduce the initial learning rate to 1/10 of the beginning, this situation will be eliminated. But because there are so many reasons that can cause this problem, I have not been able to locate the real cause.

Anyway, thank you for your enthusiastic answer!

**Comment by ZwwWayne** [comment]
@pengjizhi No problem.
Since the question seems to be answered, the issue is closed. Feel free to reopen it if you have any further questions.
